types/index.d.ts

```
@@ -1180,12 +1341,16 @@ export declare class TrackFn implements TrackFn {
 	private _exceptions;
 	private _callCount;
 	private _instances;
-	constructor(_implementation: (...args: any[]) => any);
-	track: () => (...args: any[]) => any;
+	constructor(_implementation: Function);
+	track: () => Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)